portfolio.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="portfolio.css">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	<title>Portfolio Webpage</title>
+</head>
+<body>
+	<section id="welcome-section">
+		<nav id="navbar" >
+			<a href="about.html">About</a>
+			<a href="#">Work</a>
+			<a href="#">Contact</a>
+		</nav>
+		<header class="header">
+			<h1>Hey I am Mimic </h1>
+			<p>a web developer</p>
+		</header>
+	</section>
+	<section id="projects">
+		<div class="project-tile">
+			<a href="#"></a>
+			
```

Comment:
There should not be any empty line between your code. It can make your code unclean.